### ISSUE TITLE
[Lake][CLI] Implement lake update commnd

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/pdr-backend#954]
### Background / motivation

Parent epic: [#685](https://github.com/oceanprotocol/pdr-backend/issues/685)

Check TODO - Implement "update" command

### TODOs / DoD
- [x] Implement "update" command

**Comment by KatunaNorbert** [comment]
Is the command supposed to only keep the lake up to date or should update previous rows? I suppose it should only keep the lake updated and should not take any cli params related to dates, it should use the elt logic for that,

**Comment by idiom-bytes** [comment]
```
Is the command supposed to only keep the lake up to date or should update previous rows? 
```
These are two different concepts

```
should not take any cli params related to dates
```
It should work like how we had previously.
1. ppss.yaml drives everything
2. ST + END can be passed in to override ppss.yaml lake_st and lake_end params

```
updating previous rows
```
is a mechanism inside the ETL, to update existing records when fetching new data

**Comment by idiom-bytes** [comment]
This has been implemented in PR [#956](https://github.com/oceanprotocol/pdr-backend/pull/956)

We also decided to not allow for ST + END, so it doesn't take those parameters.
This is now merged into `duckdb-integration` PR
